pkgs/applications/office/todoman/default.nix

```diff
@@ -0,0 +1,57 @@
+{ stdenv, python3, glibcLocales }:
+
+let
+  inherit (python3.pkgs) buildPythonApplication fetchPypi;
+in
+buildPythonApplication rec {
+  pname = "todoman";
+  version = "3.4.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "09441fdrwz2irsbrxnpwys51372z6rn6gnxn87p95r3fv9gmh0fw";
+  };
+
+    LOCALE_ARCHIVE = stdenv.lib.optionalString stdenv.isLinux
+      "${glibcLocales}/lib/locale/locale-archive";
+    LANG = "en_US.UTF-8";
+    LC_TYPE = "en_US.UTF-8";
+
+  buildInputs = [ glibcLocales ];
+  propagatedBuildInputs = with python3.pkgs;
+    [ atomicwrites click click-log configobj humanize icalendar parsedatetime
+      python-dateutil pyxdg tabulate urwid ];
+
+  checkInputs = with python3.pkgs;
+    [ flake8 flake8-import-order freezegun hypothesis pytest pytestrunner pytestcov ];
+
+  postInstall = ''
+    wrapProgram "$out/bin/todo" --set LOCALE_ARCHIVE ${glibcLocales}/lib/locale/locale-archive \
+    --set CHARSET=en_us.UTF-8
```

Comment:
should be `--set CHARSET en_us.UTF-8`.  (the `=` is wrong). Maybe that causes the eval error.

Also this creates two layers of wrappers (because `buildPythonApplication` already creates one) which is ugly.
A better solution may be to use `makeWrapperArgs = [ "--set LOCALE_ARCHIVE ..." "--set CHARSET ..." ];` instead of the `makeWrapper`, see other examples in nixpkgs.

Reply:
Thanks, did that. Let's see what comes out of it.